python/crun_python.c

```diff
@@ -50,8 +50,10 @@ set_error (libcrun_error_t *err)
   else
     {
       cleanup_free char *msg = NULL;
-      asprintf (&msg, "%s: %s", (*err)->msg, strerror ((*err)->status));
-      if (msg == NULL)
+      int ret;
+
+      ret = asprintf (&msg, "%s: %s", (*err)->msg, strerror ((*err)->status));
+      if (UNLIKELY (ret < 0))
```

Comment:
please indent the next line

Reply:
I think `libcrun_error_release (err)` is also needed here, similar problem as you mentioned in the next comment.

Reply:
yes, that can be a follow up patch if you'd like